invenio_rdm_records/records/api.py

```diff
@@ -290,7 +290,7 @@ class RDMMediaFileDraft(FileRecord):
     processor = DictField(clear_none=True, create_if_missing=True)
 
 
-def get_quota(record=None):
+def get_files_quota(record=None):
```

Comment:
minor: can we get away with "breaking the import" by leaving the name as it was or creating an alias such as `get_quota`? I see this is imported in [`invenio-app-rdm`](https://github.com/inveniosoftware/invenio-app-rdm/blob/7d845a427797388e3d1dd3a0ea8820d93e478275/invenio_app_rdm/records_ui/views/deposits.py#L24).

Reply:
Ah, just noticed https://github.com/inveniosoftware/invenio-app-rdm/pull/2780 😅 

Though I think it's still good that we keep the alias.